frontend/css/typography.css

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Libre+Baskerville&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+body {
+    font-family: 'Poppins', sans-serif;
+}
+
+h1 {
+    font-weight: 700;
+    font-size: 2rem;
+}
+
+h2 {
+    font-weight: 650;
```

Comment:
I think that `650` is not available as a weight, when looking at the link in the top of this file? 

Just tried setting it to 700 and I seem to get the same result - so probably the browser picks the 700 one instead? Could we set it to 600 in this case, if we'd like to differentiate it from `h1`?